pkg/bundle/bundle_test.go

```diff
@@ -201,6 +210,10 @@ func Test_Reconcile(t *testing.T) {
 				annotations[trustapi.BundleHashAnnotationKey] = fmt.Sprintf("%x", sha256.Sum256([]byte(data[*key])))
 			}
 
+			// These annotation are used for password hash sum testing
+			annotations[trustapi.BundleJksPasswdHashAnnotationKey] = "annotation will be changed during reconciliation"
+			annotations[trustapi.BundlePkcs12PasswdHashAnnotationKey] = "annotation will be changed reconciliation"
+
```

Comment:
```suggestion
```

This does not seem required. If you think I am wrong, please explain.

Reply:
Those annotation are just fake passwords hashes which are changed after the reconcile process. Of course it is possible to put something "blablabla", however I would like to keep meaningful text. Just in case.

Reply:
If we set annotations, I agree that the values should be meaningful. But do we gain anything from setting annotations here?

Reply:
Of course it is possible to use zero string `""`, however I would like to see much meaningful values here.